cmd/acra-server/common/listener.go

```diff
@@ -45,19 +47,23 @@ type SServer struct {
 	errorSignalChannel    chan os.Signal
 	restartSignalsChannel chan os.Signal
 	proxyFactory          base.ProxyFactory
+	backgroundWorkersSync sync.WaitGroup
+	stopListenersSignal   chan bool
 }
 
 // ErrWaitTimeout error indicates that server was shutdown and waited N seconds while shutting down all connections.
 var ErrWaitTimeout = errors.New("timeout")
 
 // NewServer creates new SServer.
 func NewServer(config *Config, proxyFactory base.ProxyFactory, errorChan chan os.Signal, restarChan chan os.Signal) (server *SServer, err error) {
+	stopListenersSignal := make(chan bool)
 	return &SServer{
 		config:                config,
 		connectionManager:     network.NewConnectionManager(),
 		errorSignalChannel:    errorChan,
 		restartSignalsChannel: restarChan,
 		proxyFactory:          proxyFactory,
+		stopListenersSignal:   stopListenersSignal,
```

Comment:
why not just `stopListenersSignal: make(chan bool),`?

Reply:
Yes. That's better. -1 redundant line